src/main/java/seedu/address/model/Model.java

```diff
@@ -132,6 +132,6 @@ public interface Model {
 
     void updateFilteredStudentList(Predicate<Student> predicate);
 
-
+    ObservableList<Student> getAllStudentByName(Name name);
```

Comment:
should we change this method to be called getAllStudentsByName?

Reply:
Fixed